client/layout/masterbar/publish.jsx

```diff
@@ -12,8 +13,9 @@ import SitesPopover from 'components/sites-popover';
 import paths from 'lib/paths';
 import viewport from 'lib/viewport';
 import { preload } from 'sections-preload';
+import { getSelectedSite } from 'state/ui/selectors';
 
-export default React.createClass( {
+const Publish = React.createClass( {
 	displayName: 'MasterbarItemNew',
```

Comment:
When assigning the component to a variable (or const), we can drop the `displayName`. You may probably name the const `MasterbarItemNew`. (https://facebook.github.io/react/docs/jsx-in-depth.html#the-transform)